net.pdl2ork.PdL2Ork.yml

```diff
@@ -0,0 +1,197 @@
+id: net.pdl2ork.PdL2Ork
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+default-branch: stable
+command: pd-wrapper.sh
+finish-args:
+# Display
+  - --share=ipc
+  - --socket=x11
+  #- --socket=wayland
+  - --device=dri
+# Audio
+  - --socket=pulseaudio
+  # currently disabled per a flathub maintainer's request
+  #- --system-talk-name=org.freedesktop.RealtimeKit1
+  - --filesystem=xdg-run/pipewire-0
+# Filesystem access (pd doesn't yet work with portals)
+  - --filesystem=home
+# MIDI access (Flatpak doesn't yet support exposing only MIDI devices)
+  - --device=all
+# Linux Audio plugins
+  - --env=LADSPA_PATH=/app/extensions/Plugins/ladspa:/app/lib/ladspa
+add-extensions:
+  org.freedesktop.LinuxAudio.Plugins:
+    directory: extensions/Plugins
+    version: '23.08'
+    add-ld-path: lib
+    merge-dirs: ladspa
+    subdirectories: true
+    no-autodownload: true
+
+cleanup:
+  - /include/*
+  - /lib/pd-l2ork/Makefile.am
+  - /lib/pd-l2ork/README.txt
+  - /lib/pd-l2ork/bin/todo.txt
+  - /lib/pd-l2ork/bin/nw/credits.html
+
+modules:
+- name: pd-l2ork
+  buildsystem: autotools
+  subdir: pd/src
+  config-opts: [ '--disable-portaudio', '--enable-alsa', '--disable-oss', '--enable-jack' ]
+  sources:
+  - type: git
+    url: https://github.com/pd-l2ork/pd-l2ork/
+    tag: '20231013'
+    commit: 2c3bc68b28ad55e17c07d23a715ef7ff3128068f
+  - type: patch
+    path: patch/no-rsync-during-install.patch
+
+# For Gem library
+- shared-modules/glu/glu-9.json
+# For fluid~ object
+- shared-modules/linux-audio/fluidsynth2.json
+# For LADSPA support
+- shared-modules/linux-audio/ladspa.json
+# For pdlua
+- shared-modules/lua5.3/lua-5.3.5.json
+# For the OSCx library
+- name: libtirpc
+  config-opts:
+    - --disable-gssapi
+  sources:
+    - type: archive
+      url: https://downloads.sourceforge.net/sourceforge/libtirpc/libtirpc-1.3.1.tar.bz2
+      sha256: 245895caf066bec5e3d4375942c8cb4366adad184c29c618d97f724ea309ee17
+# For bsaylor
+- shared-modules/linux-audio/fftw3f.json
+
+- name: pd-externals
+  buildsystem: simple
+  subdir: externals
+  build-commands:
+  # run Gem configure, so that iemgui can build, since it depends on it
+  # currently disabled, since we build gem below before iemgui
+  #- |
+  #  make prefix=/app \
+  #       gem_configure
+
+  # original, local testing build uses simply
+  #make ...
+  # for flathub, consider replacing make with this commented out version
+  # this needs to be tested. if it proves to work, we will be able to build
+  # with multiple cores in parallel.
+  #make -j ${FLATPAK_BUILDER_N_JOBS} ...
+
+  - |
+    make prefix=/app \
```

Comment:
Why did you remove parallel make here and below?

Reply:
only oscx seems to break on parallel

Reply:
On local builds it breaks anytime I use -j make option. The build process just randomly exits after 30-45 minutes of build time with a Code 9 (sometimes other code may show up). No other error reported.

Reply:
That's because oscx exists when using parallel, using parallel on the other ones makes it a lot faster.